src/crypto-browser.js

```diff
@@ -34,9 +35,10 @@ function webCryptoHash (type) {
       return
     }
 
-    return res.then((arrbuf) => {
-      callback(null, new Buffer(new Uint8Array(arrbuf)))
-    }).catch((err) => callback(err))
+    nodeify(
+      res.then((raw) => new Buffer(new Uint8Array(raw))),
+      callback
+    )
```

Comment:
that's how nodeify works: https://www.npmjs.com/package/nodeify It will call `callback` with `err` if it catches one, or with the result if no `err` is caught